### DISCUSSION TITLE
[Adonis 4.1] There is way to restart all event listeners in one command ?

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1954] [Help] [answered]
Hello guys,

There is way to restart all event listeners in one command ?

**Comment by thetutlage** [comment, marked as answer]
What you mean by restarting event listeners?

**Reply by wemersonrv** [reply]
forgot it my friend. I think was adonis issue related, but is from a specific lib, and i already solve it. 

Thanks for u time!
